examples/svelte/package.json

```diff
@@ -33,5 +33,6 @@
     "typescript": "^4.5.5",
     "vite": "2.9.6",
     "wait-on": "^6.0.1"
-  }
+  },
+  "bookcase-builder": {}
```

Comment:
Is this needed, if there's no configuration here?  I'd prefer if this empty config wasn't necessary.

Reply:
https://github.com/dohooo/bookcase-builder.

```shell
// package.json in the workspace directory.
// Here you can configure the project to be built. For example specify the storybook configuration directory.
// If you want to use the default configuration, you can set it to "{}". And if haven't configured it, BB will't build it.
{
  "bookcase-builder": {
    // Rename the current package name, default to package directory name.
    "basename": "other name",
    "storybook": {
      // Storybook configuration directory, default to ".storybook"
      "configDir": "./.otherStorybookDir"
    }
  }
}
```

Reply:
OK, I think it's a little confusing to have an empty config object that has meaning (causes it to be included). IMHO it would be more clear to have an explicit `include: true` (or maybe `exclude: true` if you don't want it).  Or maybe even better, a config only at the top level, with a list of subprojects to build, which would mean that each subproject does not need to have a config.  But, those are just suggestions.